app/components/widgets/forms/time-picker.js

```diff
@@ -40,9 +40,9 @@ export default Component.extend({
     this.$().calendar(merge(defaultOptions, this.options));
   },
   actions: {
-    onChange() {
+    handleFocusOut() {
       if (this.onChange) {
-        this.sendAction('onChange', this.value);
+        this.onChange(); // invoking method directly.
```

Comment:
Why to rename the onChange ?

Reply:
yeah I'll keep the same if it's an issue

Reply:
@DaKeiser How is it going ?

Reply:
Yeah, I am doing it. Facing some issues but yeah I shall submit it by sunday.